versioned_docs/version-v1.3/advanced/vgpusupport.md

```diff
@@ -54,6 +54,18 @@ The [nvidia-driver-toolkit](./addons/nvidiadrivertoolkit.md) addon needs to be e
   Once a vGPU has been assigned to a VM, it may not be possible to disable the VM until the vGPU is removed.
   :::
 
+## vGPU Devices in Rancher-Managed Clusters
+
+Ever since vGPU support was introduced in Harvester v1.3.0, Rancher has allowed attaching of vGPUs to virtual machines running on Rancher-managed Harvester clusters. However, Rancher was initially unable to track vGPU device allocation across virtual machines. You could create virtual machines with vGPU device requests that could not be accommodated, and assign vGPU profiles with unallocatable devices to guest clusters. As a result, the affected virtual machines became unschedulable and Rancher repeatedly attempted to recreate those virtual machines without success.
+
+Enhancements in Harvester v1.3.0 and v1.4.0 allow specific Rancher versions to identify the allocatable vGPU devices across all Harvester nodes. Rancher v2.8 and v2.9 display this information on the UI, and prevent the assignment of a vGPU profile when its allocatable device count does not meet the required quantity. 
+
+:::info important
+
+You must install Harvester v1.4.0 and Rancher v2.8 or v2.9 to fully leverage the improved vGPU tracking and allocation mechanism. If you upgrade Rancher but continue to use a Harvester version that does not contain the enhancements, Harvester is unable to track vGPU profiles that you assign to virtual machines using the upgraded Rancher UI.
```

Comment:
@torchiaf can you help find the exact Rancher versions that we have the feature? Thanks.

Reply:
@bk201 below the recap of all vGpu chnages:
- First implementation, https://github.com/harvester/harvester/issues/2764 - Rancher `2.9.0` and Harvester `1.3.0`
- Enhancements, https://github.com/rancher/dashboard/issues/12133 - Rancher `2.10.0`, `2.9.3`, `2.8.9` and Harvester `1.4.0` , `1.3.2`

@jillian-maroket The description looks good, please use the versions above if you want to add the references to the `patch` versions.